src/bridge.py

```diff
@@ -9,6 +11,9 @@
 import bridge_proxy
 import helpers
 
+BRIDGE_PORT = 21325
+BRIDGE_PROXY_PORT = 21326
```

Comment:
`from .bridge_proxy import PORT as BRIDGE_PROXY_PORT`?

Reply:
Nice suggestion, as we actually already import `bridge_proxy`4 lines above this